<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentSettingsWidget</class>
 <widget class="QWidget" name="AgentSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="sshAuthSockWidget" native="true">
     <layout class="QGridLayout" name="sshAuthSockOverrideLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MessageWidget" name="sshAuthSockMessageWidget" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="enableSSHAgentCheckBox">
     <property name="text">
      <string>Enable SSH Agent integration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="agentConfigPageBody">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="usePageantRadioButton">
        <property name="text">
         <string>Use Pageant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="useOpenSSHRadioButton">
        <property name="text">
         <string>Use OpenSSH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="useBothRadioButton">
        <property name="text">
         <string>Use both agents</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="agentValues">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="sshAuthSockOverrideLabel">
          <property name="text">
           <string>SSH_AUTH_SOCK override</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="sshAuthSockValueLabel">
          <property name="text">
           <string>SSH_AUTH_SOCK value</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sshAuthSockOverrideEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="sshAuthSockLabel">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="text">
           <string>(empty)</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="sshSecurityKeyProviderValueLabel">
          <property name="text">
           <string>SSH_SK_PROVIDER value</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="sshSecurityKeyProviderLabel">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="text">
           <string>(empty)</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="sshSecurityKeyProviderOverrideLabel">
          <property name="text">
           <string>SSH_SK_PROVIDER override</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="sshSecurityKeyProviderOverrideEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>gui/MessageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
